pkg/tuned/controller.go

```diff
@@ -758,9 +906,33 @@ func GetBootcmdline() (string, error) {
 	return responseString, nil
 }
 
+// changeSyncerPostReloadOrRestart syncronizes the daemon status after a node restart or a tuned reload
+func (c *Controller) changeSyncerPostReloadOrRestart(change Change) {
+	klog.V(2).Infof("changeSyncerPostReloadOrRestart(%s)", change.String())
+	defer klog.V(2).Infof("changeSyncerPostReloadOrRestart(%s) done", change.String())
+
+	if !change.nodeRestart && !change.tunedReload {
```

Comment:
Do we intend to synchronize *only* when the change is not nodeRestart and not tunedReload?  If so, I believe we want to expand the comment a little.

Reply:
for the deferred update yes, this is the only point in time on which they should be applied.